.github/docs/release.md

```diff
@@ -93,20 +93,23 @@ This checklist will prepare `v1.3.0` from `v1.2.0`. Adjust your version numbers
            gh workflow run generate-measurements.yml --ref release/v$minor -F osImage=v$ver -F isDebugImage=false -F signMeasurements=true
         ```
 
-    13. Create a new tag on this release branch
+    13. Create a new tag on this release branch.
 
         ```sh
         git tag v$ver
         git tags --push
         ```
 
-        * Run [Release CLI](https://github.com/edgelesssys/constellation/actions/workflows/release-cli.yml) action on the tag
+    14. Update expected measurements in [`measurements.go`](/internal/attestation/measurements/measurements.go) using the generated measurements from step 12 and **push your changes**.
```

Comment:
I think we need to swap 13 & 14, or else the commit to `measurements.go` will not be included in the tag, and therefore the `release-cli.yml` pipeline run won't include it.

Reply:
You're right.
Had creating the tag confused with creating the release branch